example/src/main/scala/example/config/AppConfig.scala

```diff
@@ -16,12 +16,11 @@
 
 package example.config
 
-import zio.config.ConfigDescriptor
-import zio.config.magnolia.DeriveConfigDescriptor
+import zio.config.magnolia.descriptor
 import zio.redis.RedisConfig
 
 final case class AppConfig(redis: RedisConfig, server: ServerConfig)
 
 object AppConfig {
-  val descriptor: ConfigDescriptor[AppConfig] = DeriveConfigDescriptor.descriptor[AppConfig]
+  val confDescriptor: _root_.zio.config.ConfigDescriptor[AppConfig] = descriptor[AppConfig]
```

Comment:
What happened here?

Reply:
descriptor is now the top-level method instead of DeriveConfigDescriptor's method. And i will delete package prefix